pyproj/geod.py

```diff
@@ -240,9 +240,17 @@ def __init__(self, initstring: Optional[str] = None, **kwargs) -> None:
         )
 
     def fwd(  # pylint: disable=invalid-name
-        self, lons: Any, lats: Any, az: Any, dist: Any, radians: bool = False
+        self,
+        lons: Any,
+        lats: Any,
+        az: Any,
+        dist: Any,
+        radians: bool = False,
+        return_back_azimuth: bool = True,
```

Comment:
I have been thinking about this and am thinking it would be a good idea to warn users of
the upcoming change.

This change can initially be released with:
```
return_back_azimuth: Optional[bool] = None,
```
and 
```python
if return_back_azimuth is None:
    return_back_azimuth = False
    warnings.warn(
        "The forward azimuth is currently being returned by default. "
        "In the future, the back azimuth will be returned. To opt-in "
        "to the new behavior, set return_back_azimuth=True. "
        "Otherwise, to avoid breaking changes, set return_back_azimuth=False "
        "for pyproj 3.5+."
    )
```
Then, in a future release, the default can be changed to True.

Thoughts?

Reply:
Another idea is to change the default and raise a warning similar to the above if the user has not selected True or False so they know that they are getting the back azimuth now.

Reply:
> Another idea is to change the default and raise a warning similar to the above if the user has not selected True or False so they know that they are getting the back azimuth now.

Yes, sounds good.

Reply:
Great. @idanmiara, is this something you have time to add to this PR?

Reply:
yes. added and rebased.